### PR TITLE
Nodejs week2

### PR DESCRIPTION
**Pull request by ShravyaGurmitkal**

### CODE REVIEW

File: src/backend/api/reservations.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//Returns all reservations
+router.get("/", async (request, response) => {
+  try {
+    const allReservations = await knex("reservation");
```
Comment (itsgauravsaxena) [inline src/backend/api/reservations.js:8]: maybe it works but this seems more readable format
knex
  .from('books')
  .select('title', 'author', 'year')

But this is just my personal opinion :)

File: src/backend/api/reservations.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+//Updates the reservation by id
+router.put("/:id", async (request, response) => {
+  try {
+    const updateReservation = request.body
+    if(Object.keys(updateReservation).length === 0) {
+      response.status(400).send('Please send the fields to update')
+    } else {
+      const updatedId = await knex("reservation").where({'id': request.params.id}).update(updateReservation);
+      const allReservations = await knex("reservation");
+      (updatedId !== 0) ? response.json({ message: "Updated reservation", updateStatus: updatedId, reservations: allReservations }) : response.status(404).json({ message: "No reservation is available with this id", updateStatus: updatedId, reservations: allReservations })
```
Comment (itsgauravsaxena) [inline src/backend/api/reservations.js:57]: length of the line is too long. Feel free to use new line characters to make it more readable :)

**Review by itsgauravsaxena — APPROVED** [review]

File: src/backend/api/meals.js [hunk 3]
```diff
@@ -2,14 +2,77 @@ const express = require("express");
...
+//Updates the meal by id
+router.put("/:id", async (request, response) => {
+  try {
+    const updateMeal = request.body;
+    if(Object.keys(updateMeal).length === 0) {
+      response.status(400).send('Please send the fields to update')
+    } else {
+      const updatedId = await knex("meal").where({'id': request.params.id}).update(updateMeal);
+      const allMeals = await knex("meal");
+      (updatedId !== 0) ? response.json({ message: "Updated meal", updateStatus: updatedId, meals: allMeals }) : response.status(404).json({ message: "No meal is available with this id", updateStatus: updatedId, meals: allMeals })
```
Comment (RZhyvitskyi) [inline src/backend/api/meals.js:58]: A bit hard to read in one line. Can be nice to use normal if - else statement.

**Review by RZhyvitskyi — APPROVED** [review]
I'm trying to use ternary operators only for short expressions. But the rest is good.

**Pull request merged**